modules/data-entry/src/main/frontend/src/dataHomepage/Filters.jsx

```diff
@@ -41,13 +41,13 @@ const ALL_QUESTIONNAIRES_URL = "/Questionnaires.deep.json";
 const FILTER_URL = "/Questionnaires.filters";
 
 function Filters(props) {
-  const { classes, disabled, onChangeFilters, questionnaire, filtersJsonString } = props;
+  const { classes, disabled, onChangeFilters, questionnaire, filtersJsonString, statisticFilters, parentHandler, statisticFiltersValue } = props;
   // Filters, as displayed in the dialog, and filters as actually saved
-  const [editingFilters, setEditingFilters] = useState([]);
+  const [editingFilters, setEditingFilters] = useState([]); // TODO: existing
```

Comment:
Is the loading of filters from the URL, which we did recently implement, considered a fix for this TODO?

```suggestion
  const [editingFilters, setEditingFilters] = useState([]);
```